setup.py

```diff
@@ -7,9 +7,9 @@
 
 setuptools.setup(
     name="s2aff",
-    version="0.11",
+    version="0.12",
```

Comment:
This is already 0.12 in main https://github.com/allenai/S2AFF/blob/main/setup.py, so should prob be 13

Reply:
Ooops, fixed.